src/components/atoms/Link/index.tsx

```diff
@@ -0,0 +1,46 @@
+import cx from 'classnames';
+import Link from 'next/link';
+import React from 'react';
+
+export type LinkProps = {
+  href: string;
+  title?: string;
+  bgColor?: 'none' | 'primary' | 'white' | 'gray';
+  textSize?: 'small' | 'normal' | 'large';
+  textColor?: 'primary' | 'white' | 'gray';
+};
+
+const LinkButton: React.FC<LinkProps> = ({
+  href,
```

Comment:
it's good add more props.
Example `externalLink` and if true, add a external icon and target blank in `<a>` tag.